crates/sui-types/src/query.rs

```diff
@@ -49,13 +47,8 @@ pub enum EventQuery {
         /// the event struct name type, e.g. `0x2::devnet_nft::MintNFTEvent` or `0x2::SUI::test_foo<address, vector<u8>>` with type params
         String,
     ),
-    EventType(EventType),
```

Comment:
not in this PR but EventQuery like EventFilter, seems to me more of a JSON RPC concept rather than sui-types here?

Reply:
I am working with @666lcz to combine it In this PR